docs/quickstart.md

```diff
@@ -1,27 +1,20 @@
 # Quickstart
 
-## Set up Prisma
+## Setup
 
-1) Create a new project with Go modules
```

Comment:
Ok! I was thinking that since go mod is now the default, we could expect the users to have this in place, but I agree that it can't hurt if we add these instructions as a first step.